src/scheduler/data_types.h

```diff
@@ -194,21 +194,21 @@ struct th_data_nd_eligible
 
 struct th_data_dup_nd_info
 {
+	unsigned int err:1;
```

Comment:
I'd call this error.  Currently the only time we have a variable called 'err' it is a schd_error.  You even call a member err later that is of type schd_error.